core-bundle/src/Image/PictureFactory.php

```diff
@@ -120,12 +120,11 @@ public function create($path, $size = null, ResizeOptions $options = null): Pict
         if ($size instanceof PictureConfiguration) {
             $config = $size;
         } else {
-            [$config, $attributes, $options] = $this->createConfig($size);
+            [$config, $attributes, $configOptions] = $this->createConfig($size);
         }
 
-        if (null === $options) {
-            $options = new ResizeOptions();
-        }
+        // Always prefer options passed to this function
+        $options = $options ?? $configOptions ?? new ResizeOptions();
```

Comment:
```suggestion
        $options ??= $configOptions ?? new ResizeOptions();
```

Reply:
Unfortunately this is only available in PHP >= 7.4.

Reply:
☹️